packages/primitives/image/src/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ImageResizeMode } from 'react-native'
+
+type TObjectFitProperty = 'fill' | 'contain' | 'cover' | 'none' | 'scale-down'
```

Comment:
there must be a better way.... something like.... CSSProperties.ObjectFitProperty
IDK :/
couldn't figure it out

Reply:
we can't have 2 different types for web and native – the whole idea of primitive is to provide a single universal interface that works _the same_ for all the platforms.

let's make it only `'cover' | 'contain'` for both web and native for now:

```ts
export type TImage = {
  id?: string,
  alt?: string,
  source: string,
  height: number,
  width: number,
  borderRadius?: number,
  resizeMode?: 'cover' | 'contain'
}
```

Reply:
💩
makes sense!!!!
fixed!